src/components/data/Scroll.js

```diff
@@ -0,0 +1,42 @@
+import classnames from 'classnames';
+
+import { downcastRef } from '../../util/typing';
+
+/**
+ *
+ * @typedef {import('../../types').PresentationalProps} CommonProps
+ * @typedef {import('preact').JSX.HTMLAttributes<HTMLDivElement>} HTMLDivAttributes
+ *
+ * @typedef ScrollProps
+ * @prop {'raised'|'flat'} [variant='raised'] - Render with scroll-hinting
+ *   shadows ('raised') or without ('flat')
```

Comment:
Using the flat variant without a scroll-ability hint may be confusing unless there is some other affordance to indicate scrollability (assuming an OS that doesn't always render scrollbars). Do you know what the use cases we currently have for `flat` are?

Reply:
Fair question. I can't cite a current use case. But I could foresee potential use cases:

* A case in which we are unable to, for some reason, set a background color on the scrollable content, or it needs to be non-white. This would break the shadowing.
* A case in which the alignment of scrollable content was such that the shadows were mis-aligned.

However, arguably, that could be handled by not using this component in those cases. Hmmmm...

Reply:
After some reflection, I think I'd like to leave this "shadow kill switch" in place for the time being, and leave myself a note to revisit it at the end of this current project. One reason is that I know that `Table` will make use of scroll components, which may involve a use case in which shadows are problematic. 

I'm sort of like an airline pilot, a gut dislike of having any automated system that can't be disabled in an emergency :P

I added a task on the project board to revisit this.